README.md

```diff
@@ -58,11 +87,54 @@ end
 
 winner  
 ```
+
+If you run your tests now, you should be passing. 
+
+### A Closer Look
+
 Let's break this down: 
 
-* We iterate through the hash using `.each`. We chose `.each` instead of collect because we don't want to collect the key/value pair that contains the winner, just the *name* of the winner. With `.each`, we have the control we need to simply grab the winner's name and set it equal to a variable that we can return later on. 
+* We iterate through the hash using `#each`. We chose `#each` instead of collect because we don't want to collect the key/value pair that contains the winner, just the *name* of the winner. With `#each`, we have the control we need to simply grab the winner's name and set it equal to a variable that we can return later on. 
```

Comment:
If they haven't BTW. Just say "you may have seen online the `#collect` method. We don't use that because blah blah blah. If you have never seen the `#collect` method don't worry about it, we'll get there real soon!" I like this explanation and I don't want to lose it